packages/test-runner-visual-regression/src/visualDiffCommand.ts

```diff
@@ -79,22 +103,28 @@ export async function visualDiffCommand(
     };
   }
 
-  const { diffImage, diffPercentage, error } = await options.getImageDiff({
+  const result = await options.getImageDiff({
     name,
     baselineImage,
     image,
     options: options.diffOptions,
   });
 
+  const { error, diffImage } = result;
+
   if (error) {
     // The diff has failed, be sure to save the new image.
     await saveFailed();
     await saveDiff();
 
-    throw new VisualRegressionError(error);
```

Comment:
Throwing an error here, rather than relying on the usual returned object, created lots of noise in the console log since you get a stack trace spat out separately from the usual test failure output: 

```
VisualRegressionError: Screenshot is not the same width and height as the baseline. Baseline: { width: 752, height: 152 } Screenshot: { width: 752, height: 155 }
    at Object.visualDiffCommand (/Users/blah/modern-web/packages/test-runner-visual-regression/dist/visualDiffCommand.js:89:15)
    at async TestRunnerApiPlugin._onCommand (/Users/blah/design-system/packages/components/node_modules/@web/test-runner-core/dist/server/plugins/api/testRunnerApiPlugin.js:145:32)
```

So I made a slight tweak to use the regular mechanism for reporting failure. Now the output is cleaner and more consistent.

Hope that's ok? Couldn't see any real reason why an error is thrown here for failure, but every other failure case returns an object. Maybe i'm missing something!